apps/mobile/index.js

```diff
@@ -32,11 +32,15 @@ class App extends React.Component<Props, State> {
       'orbit-icons': OrbitIcons,
     });
 
-  async componentDidMount() {
-    await this.loadFonts();
+  afterFontLoaded = () => {
     this.setState({ fontsLoaded: true }, () => {
       Expo.SplashScreen.hide();
     });
+  };
+
+  async componentDidMount() {
```

Comment:
Maybe I am just damaged from having used `airbnb-eslint-config` in the past, but my OCD tells me `componenentDidMount` should be the first function defined an a class.

Reply:
And, there is an eslint-rule for it, `react/sort-comp`, if we want to enforce it 😊

Reply:
👍 thanks I will check that rule

Reply:
done